### ISSUE TITLE
Disabled token can be enabled despite Active Tokens Count exceeding the value set in Policy

### ISSUE THREAD

**Issue by mail2hemant** [privacyidea/privacyidea#2215]
### Top-level intent

What did you try to achieve?
Token policy is set to have only 01 Active TOTP Token for the user. Checking validation if system prevents users to have multiple Active TOTP tokens despite policy set to allow only 01 active token.

### Steps to reproduce

1. Login as admin, Create policy to allow max 01 Active TOTP token and Max 10 tokens.
2. Enroll TOTP token for a user, disable it.
3. Enroll another TOTP token for same user and its a active/enabled state.
4. Try to enroll another TOTP token for the same user and system will prevent the enrollment.
5. Now try to Enable the disabled token for the same user and system doesn't prevent enabling.

### Expected outcome

What do you think, how the system should have worked?
System should have checked the policy and prevented user/admin from enabling the token, if the user already has max active tokens for token type.

### Actual outcome
**Bug** - Now user has 2 active TOTP tokens despite policy only allowing max 1 Active TOTP token.

What did actually happen?

### Configuration

* **privacyIDEA version**:
* **Installation method**: (from Ubuntu packages, github, PyPI, ...)
* **Python version**:
* **Operating system**:
* **Webserver**:
* **Token database**: (MySQL, PostgreSQL, ...)

### Log file

**Set PI_LOGLEVEL = logging.DEBUG in pi.cfg and take a look at the privacyidea.log!**
**If appropriate, attach the log file or paste relevant portions.**

**Comment by cornelinux** [comment]
The decorator ``check_max_token_user`` checks all max_token-policies, also the ``ttype_max_active_token_per_user`` policies.

The decorator actually decorates the API endpoints ``/token/init``, ``/token/assign`` and ``/token/enable``. 

If the tokentype would not be specified, it determines the tokentype by the serial.
https://github.com/privacyidea/privacyidea/blob/97a4d95ff3a41156ac6fb17a5e57720948d05b76/privacyidea/api/lib/prepolicy.py#L623
So basically it should work.

The problem seems to be, that when *enabling* a token, the request has no user_obj, and thus the policy is not verified:
https://github.com/privacyidea/privacyidea/blob/97a4d95ff3a41156ac6fb17a5e57720948d05b76/privacyidea/api/lib/prepolicy.py#L620
